MessengerOnFileForLinux/TerminalFunctionality/test/TerminalFunctionalityTestCore.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <gtest/gtest.h>
+#include <TerminalFunctionality.hpp>
```

Comment:
enter miedzy naszymi a resztą includów

Reply:
Nie zgodzę się. Ja dzielę na dwa: te które coś wnoszą i te które każdy wie, że muszą tu być. Gdyby to np. był PurgeMessage.hpp to byłby po enterze, ale tutaj wiemy, że oba na 100% spotkamy nie czytając nawet includów.

Reply:
Do przegadania, bo róznie jest klasach